server/command/install.go

```diff
@@ -78,7 +78,7 @@ func (s *service) executeInstall(params *params) (*model.CommandResponse, error)
 	}, nil
 }
 
-func (s *service) executeExperimentalInstall(params *params) (*model.CommandResponse, error) {
+func (s *service) executeProvision(params *params) (*model.CommandResponse, error) {
```

Comment:
Just to be clear - need to document who is executing this command and at what step in the "deploy new app" process.

Reply:
I too am a little confused by the long-term intention of this command; is it intended for developers?

Reply:
This command is for developers/testing. We will not use it in production.

Reply:
I still miss smth. = ) - where do we call the AWS function provisioning in production?

Reply:
@catalintomai we still need to implement a separate provisioning "script" that would create the lambdas/S3 and generate the manifest for an App.